.circleci/create_circleci_config.py

```diff
@@ -310,6 +308,7 @@ def job_name(self):
         "pip install 'git+https://github.com/facebookresearch/detectron2.git'",
         "sudo apt install tesseract-ocr",
         "pip install pytesseract",
+        "pip install natten",
     ],
     tests_to_run="tests/models/*layoutlmv*",
```

Comment:
@sgugger Just one last question. Do you prefer if the tests related to NAT/DiNAT are added under a new layoutlmv directory, or do I just manually include the two subdirectories (`tests/models/*nat`) to `tests_to_run`?

Reply:
You should manually add the two subdirectories here, please!